pkg/selinux/selinux-client.js

```diff
@@ -147,11 +166,26 @@ export function init(statusChangedCallback) {
                         if (items.length === 2 && items[1] == "-D")
                             return result;
 
-                        if (manageditems.indexOf(items[0]) < 0)
+                        if (manageditems.indexOf(items[0]) < 0) {
+                            if (items[0] in shell_rules)
+                                shell_rules[items[0]].push("    semanage " + mod);
+                            else
+                                shell_rules[items[0]] = ["    semanage " + mod];
                             result.push(mod);
+                        }
                         return result;
                     }, [])));
 
+                    // Create shell rule for every ansible group separatelly
+                    Object.keys(shell_rules).forEach(t => {
```

Comment:
Let's sort the keys, so that we get a predictable ordering.

Reply:
Order is always the same (and follows order in which they are listen in shell script).
It is thanks to semanage exporting it in the same order thanks to [this](https://github.com/SELinuxProject/selinux/blob/master/python/semanage/semanage#L783) and since ES2015 (non-numeric) keys in object keep order in which they were inserted.

Reply:
Interesting, I didn't know about the latter (keys in objects keeping insert order)